packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -199,6 +199,36 @@ class GuildBanManager extends CachedManager {
     await this.client.rest.delete(Routes.guildBan(this.guild.id, id), { reason });
     return this.client.users.resolve(user);
   }
+
+  /**
+   * Ban up to 200 users from a guild, and optionally delete previous messages sent by the banned users. Requires
+   * both the `BAN_MEMBERS` and `MANAGE_GUILD` permissions.
+   * @param {BanBulkOptions} options The options for bulk banning users
+   * @returns {Promise<BanBulkResult>} Returns an object with `bannedUsers` key containing the IDs of the banned users
+   * and the key `failedUsers` with the IDs that could not be banned or were already banned.
+   * @example
+   * // Bulk ban users
+   * guild.bans.bulk({ users: ['84484653687267328'], deleteMessageSeconds: 604_800 })
```

Comment:
```suggestion
   * guild.bans.bulk({ users: ['84484653687267328'], deleteMessageSeconds: 24 * 60 * 60 })
```

Magic numbers arent very nice